config/default.yaml

```diff
@@ -120,6 +120,14 @@ executor:
 
         # Launcher container tag to use
         launchVersion: stable
+    jenkins:
```

Comment:
Could probably have this executor commented out, much like for gitlab and bitbucket
It should also work fine if it isn't